### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by jfst** [NREL/floris#15]
**Bug description**
The calculate_wake() function produces

 different result even though the input is the same.

**To Reproduce**
Run the code in the Additional context section. The input file is the one provided in the example directory of the floris package.

**Expected behavior**
The total output power differ for the same input wind direction (-270) when calc_wake() is called between with another wind direction (-90) .

Wind speed/dir:  8.0 / -270
Power:  **4.470527954624492**

Wind speed/dir:  8.0 / -90
Power:  5.276564072664706

Wind speed/dir:  8.0 / -270
Power:  **5.276564072664707**

Wind speed/dir:  8.0 / -90
Power:  5.276564072664706

**Screenshots, if applicable**
<Add screenshots to help explain your problem.>

**Floris Version**
v1.1.2

**System Information (please complete the following information):** 
 - OS: Win10
 - Library versions
   * matplotlib: 3.0.3
   * numpy: 1.16.3
   * pytest: 4.3.1
   * scipy: 1.1.0
   * Sphinx: 1.8.5

**Additional context**
`import numpy as np
import floris.tools as wfct

fi = wfct.floris_utilities.FlorisInterface('example_input.json')
print()

fi.reinitialize_flow_field(wind_direction = 0)
print('Wind speed/dir: ', fi.floris.farm.wind_speed, '/', fi.floris.farm.wind_direction)
fi.calculate_wake()
print('Power: ', np.sum(fi.get_turbine_power())/1e6)
print()

fi.reinitialize_flow_field(wind_direction = 180)
print('Wind speed/dir: ', fi.floris.farm.wind_speed, '/', fi.floris.farm.wind_direction)
fi.calculate_wake()
print('Power: ', np.sum(fi.get_turbine_power())/1e6)
print()

fi.reinitialize_flow_field(wind_direction = 0)
print('Wind speed/dir: ', fi.floris.farm.wind_speed, '/', fi.floris.farm.wind_direction)
fi.calculate_wake()
print('Power: ', np.sum(fi.get_turbine_power())/1e6)
print()

fi.reinitialize_flow_field(wind_direction = 180)
print('Wind speed/dir: ', fi.floris.farm.wind_speed, '/', fi.floris.farm.wind_direction)
fi.calculate_wake()
print('Power: ', np.sum(fi.get_turbine_power())/1e6)
print()`

**Comment by bayc** [comment]
Hello @jfst . Thank you for contributing to the FLORIS project! We believe we have found the solution to the issue you have identified and I have opened a pull request to fix it (https://github.com/NREL/floris/pull/16). 

I've included a detailed description of the issue in the pull request, along with plots and qualitative results demonstrating the power outputs before and after the fix, but in brief, the turbine's turbulence intensity was not being correctly reinitialized when `reinitialize_flow_filed()` was being called. Feel free to add to the conversation there if you feel anything  wasn't addressed, and thanks again for submitting this bug report.
